src/components/modal/modal.module.js

```diff
@@ -159,8 +161,8 @@
    * 		</blt-modal>  
    * 		
    * 		<div class="btn-row-left">
+   *    		 <button class="btn-solid-submit" blt-open="simpleModal">X-small Modal</button>
```

Comment:
Small thing, but the spacing here may also appear offset in the 'HTML' tab of the runnable sample.